### ISSUE TITLE
Insaflu-server build is failing

### ISSUE THREAD

**Issue by jakobkolb** [INSaFLU/docker#17]
During the docker build, the ISaFLU base installation is failing due to a dependency conflict:

```
#0 1120.5 Collecting beautifulsoup4==4.11.1
#0 1120.6   Downloading beautifulsoup4-4.11.1-py3-none-any.whl (128 kB)
#0 1120.6      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.2/128.2 kB 6.0 MB/s eta 0:00:00
#0 1120.6 ERROR: Ignored the following versions that require a different python version: 0.7 Requires-Python >=3.6, <3.7; 0.8 Requires-Python >=3.6, <3.7; 1.6.2 Requires-Python >=3.7,<3.10; 1.6.3 Requires-Python >=3.7,<3.10; 1.7.0 Requires-Python >=3.7,<3.10; 1.7.0rc1 Requires-Python >=3.7,<3.10; 1.7.0rc2 Requires-Python >=3.7,<3.10; 1.7.1 Requires-Python >=3.7,<3.10
#0 1120.6 ERROR: Could not find a version that satisfies the requirement dataclasses==0.8 (from versions: 0.1, 0.2, 0.3, 0.4, 0.5, 0.6)
#0 1120.6 ERROR: No matching distribution found for dataclasses==0.8
#0 1120.9 Error installing INSaFLU base
```

**Comment by jakobkolb** [comment]
tbh. due to the way you are building the image, debugging this build is immensely time consuming and painful. Have you considered providing a pre built image for the insaflu-server Container?

**Comment by dsobral** [comment]
I agree that the build is very complex, and debugging it is cumbersome. In particular the installation of the TELEVIR part (which is already a separate docker service from INSaFLU) is very time consuming and prone to connection and other issues. 

We are considering a pre built image, but we have to check all software/data licenses and also see a way to balance image size versus installation ease of use.

We welcome any suggestions to make it easier to make INSaFLU + Televir available to a wider audience.

**Comment by jakobkolb** [comment]
I have two suggestions

First: Provide prebuild images for insaflu and televir on Dockerhub,
Second: Add build tests with github actions to your repository to make sure, the builds are actually working in an environment that is not your personal machines.

We would be happy to work with InsaFLU at Robert Koch Institute, provided you can do the two things above.

Many Thanks!

**Comment by dsobral** [comment]
Dear @jakobkolb 

We have now prebuild images for INSaFLU. 
At least this component should be much easier to install.

Currently there is no prebuilt image for Televir mostly due to its size. 
We hope to provide at some point some alternatives for easier installation of the Televir component.

There is currently no automated github actions due to the complexity and size of the current system.

Daniel
INSaFLU Team
